ansible_collections/arista/avd/requirements.txt

```diff
@@ -9,3 +9,4 @@ PyYAML>=5.4.1
 md-toc>=7.1.0
 deepmerge>=1.1.0
 cryptography>=38.0.4
+anta==0.7.2
```

Comment:
```suggestion
anta==0.7.2 # Pinned to specific version until anta reached 1.0.0 and follows semver.
```

Reply:
addressed in https://github.com/aristanetworks/ansible-avd/commit/de04df313f6991308683742ffa0528b81d5d8ec9

our verify-requirement does not understand comments after the line so I put it above. will need to log an issue for this probably